.github/workflows/workflow.yaml

```diff
@@ -78,6 +78,14 @@ jobs:
       - name: Run script
         run: |
           make require-suites
+  require_sync:
+    runs-on: ubuntu-latest
+    name: "Require a sync.yaml file and metadata.gatekeeper.sh/requiresSyncData annotation for every template.yaml using data.inventory"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run script
+        run: |
+          make require-sync
```

Comment:
it's good that we are checking here. should we also add validation in Gatekeeper to write an error in status if the sync resource is missing the required data in `metadata.gatekeeper.sh/requiresSyncData`?

Reply:
Thanks @ritazh! - Absolutely, however that scope of work is beyond what I can include in this particular PR.  Should we track as a feature request issue for future contribution?  @maxsmythe @ekitson

Reply:
> or is missing in data.inventory

I don't think this can be reliably determined: is it missing because it's not being synced or because there is nothing to sync?

> write an error in status if the required data specified in metadata.gatekeeper.sh/requiresSyncData is not part of the sync resource

This makes sense to do, but is non-trivial. I'd file an issue in G8r.

Reply:
Opened: https://github.com/open-policy-agent/gatekeeper/issues/2393